internal/migrations/20250115120000_add_performance_indexes.go

```diff
@@ -0,0 +1,28 @@
+package migrations
+
+import (
+	gormigrate "github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func addPerformanceIndexes() *gormigrate.Migration {
```

Comment:
We already have an index for clusters here: https://github.com/openshift/assisted-service/blob/master/internal/migrations/20250728120000_add_host_by_cluster_id_index.go#L10, I think it would be good to be consistent for the nameing of the index, the migration function, and the file name.